.github/workflows/dapr.yml

```diff
@@ -173,3 +180,33 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Package Helm chart
```

Comment:
Let's make it simpler by adding [new workflow job](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs) 

in `publish` job
1. Generate helm package inside `publish` job.
2. Upload this helm chart package to artifact.

in `release-helm` (new job, please find the better name :) )
1. Check out only helm repo
2. Download helm chart package from artifact
3. run git command to push the package to the repo. 

Then,

We do not need to clone multiple repo into the single vm.